### DISCUSSION TITLE
is not authorized to perform `ses:SendRawEmail' on resource

### DISCUSSION THREAD

**Discussion by Rinki1479** [symfony/symfony#51463] [Q&A]
### Symfony version(s) affected

6.3

### Description

When i am sending email getting this error 

`arn:aws:ses :identity/abc@gmail.com' (code AccessDeniedException). {"userId":1,"exception":"[object] (Symfony\\Component\\Mailer\\Exception\\HttpTransportException(code: 403): Unable to send an email: User arn:aws:iam: is not authorized to perform 'ses:SendRawEmail' on resource (code AccessDeniedException). at /var/app/current/vendor/symfony/amazon-mailer/Transport/SesHttpAsyncAwsTransport.php:63)`
[stacktrace]

### How to reproduce

```php
 public function sendEmail(Request $request)
    {   
        $accessKey = urlencode(env('AWS_ACCESS_KEY_ID'));
        $secretKey = urlencode(env('AWS_SECRET_ACCESS_KEY'));

    
        $transport = Transport::fromDsn('ses+https://'.$accessKey.':'.$secretKey.'@default');

        $mailer = new Mailer($transport);

        $email = (new Email())
            ->from('abc@gmail.com')
            ->to('abc@gmail.com')
            ->subject('Test Mail')
            ->text('Sending emails');

        $mailer->send($email);

        dd($mailer);

    }
```

**Comment by stof** [comment]
Well, fix your IAM permissions in the AWS config so that your credentials are allowed to use this API.

**Comment by Rinki1479** [comment]
I don't have IAM permissions in the AWS config but another project my credentials is working How ?

**Reply by Rinki1479** [reply]
please reply

Thanks
